pkg/apicapi/apicapi.go

```diff
@@ -264,6 +270,8 @@ func (conn *ApicConnection) restart() {
 		conn.log.Debug("Restarting connection")
 		close(conn.restartCh)
 		conn.restartCh = nil
+		conn.log.Debug("Resetting deltaQueue")
+		conn.deltaQueue = nil
```

Comment:
Hate to be pedantic - In the commit message could you add some context/details regarding the issue we are noticing and how we expect resetting the queue to handle that?